docs/components/operate/userguide/resolve-incidents-update-variables.md

```diff
@@ -9,6 +9,8 @@ import TabItem from "@theme/TabItem";
 
 Every process instance created for the [`order-process.bpmn`](./assets/order-process.bpmn) process model requires an `orderValue` so the XOR gateway evaluation will happen properly.
 
+<!--- Maybe I'm totally just missing something here, but where was this model from? Is it in previous documentation? --->
+
```

Comment:
From what I can see this has alway been here. It is the model that needs to be deployed and used by the examples.